src/scripts/components/widgets/FilesUploader.js

```diff
@@ -131,58 +123,58 @@ class FilesUploader extends Component<Props, Object> {
         file: file,
         fileName: file.name + ' | ' + file.size + 'bytes'
       })
+
+      this.props.history.push(FILESUPLOADER_PATH_TO)
     } else {
       this.props.checkUserWallet()
     }
   }
 
   onDrag (e: Object) {
-    // If wallet not secure open the modal
-    if (this.props.isWalletSecured) {
-      const status = e.type
-      let klass = ''
-      if (status === 'dragenter' || status === 'mouseover') {
-        klass = 'dragenter'
-      } else if (status === 'drop') {
-        klass = 'drop'
-      } else {
-        klass = ''
-      }
-      this.setState({
-        dragClass: klass
-      })
+    const status = e.type
+    let klass = ''
+    if (status === 'dragenter' || status === 'mouseover') {
+      klass = 'dragenter'
+    } else if (status === 'drop') {
+      klass = 'drop'
     } else {
-      this.props.checkUserWallet()
```

Comment:
why do remove this control? without that the user can upload even if is not logged in

Reply:
hum. I don't know why. Let me fix